### PR TITLE
arm64

### PR DESCRIPTION
**Pull request by andersthorsen**
Hello!

I tried to use this on arm64 and it failed to pull it from docker hub as no arm64 build exist so I built one myself with these modifications. As no arm64 version seems to exist of the software, I had to add armhf support inside the arm64 docker image used.

As for the dpgk commands I'm not sure where the best place for them would be.

### CODE REVIEW

**Comment by Thom-x** [comment]
For s6 we should use aarch64 version.

> s6-overlay-aarch64.tar.gz - includes binaries for 64-bit ARM platforms.

File: build/fr24feed.sh [hunk 1]
```diff
@@ -5,6 +5,15 @@ case $arch in
     url=https://repo-feed.flightradar24.com/rpi_binaries/fr24feed_${FR24FEED_ARMHF_VERSION}_armhf.tgz # force version 1.0.25-3 because of broken version for rpi
     dirname=fr24feed_armhf
     ;;
+  arm64)
+    url=https://repo-feed.flightradar24.com/rpi_binaries/fr24feed_${FR24FEED_ARMHF_VERSION}_armhf.tgz # force version 1.0.25-3 because of broken version for rpi
+    dirname=fr24feed_armhf
+    dpkg --add-architecture armhf
+    apt update
+    apt dist-upgrade
```
Comment (Thom-x) [inline build/fr24feed.sh:13]: Is it necessary to do dist upgrade ?

File: build/fr24feed.sh [hunk 2]
```diff
@@ -5,6 +5,15 @@ case $arch in
...
+    apt install libc6:armhf libstdc++6:armhf libusb-1.0-0:armhf
```
Comment (Thom-x) [inline build/fr24feed.sh:14]: Could you also remove cache to have a smaller image ?

```
apt-get install -y \
    xxx && \
    rm -rf /var/lib/apt/lists/*
```

**Review by Thom-x — CHANGES_REQUESTED** [review]

File: build/fr24feed.sh (hunk 2 above)
Comment (andersthorsen) [reply build/fr24feed.sh:13]: Done

File: build/fr24feed.sh (hunk 1 above)
Comment (andersthorsen) [reply build/fr24feed.sh]: Apparently not as it worked after removing that step.

**Comment by andersthorsen** [comment]
I think all comments has been addressed. Please let me know if there are additional comments.

Apologies for the slow reply but I ran into unrelated issues when trying to re-run my build of those adjustents.

**Review by Thom-x — APPROVED** [review]

**Comment by Thom-x** [comment]
We should also add the arch to build in `.github/workflows/xxx` files.

**Comment by Thom-x** [comment]
@deploy

**Comment by Thom-x** [comment]
You can test it with the tag `PR-54` and tell me if it's working well 👍.

**Comment by andersthorsen** [comment]
No arm64 image listed at https://hub.docker.com/r/thomx/fr24feed-piaware/tags?page=1&ordering=last_updated

Do you think something is wrong with the workflow files? Or was the deployment done using the pre-merge workflow files.

**Comment by Thom-x** [comment]
I think it was fine with the pre merged files. I will accept the PR and we will see.

**Comment by andersthorsen** [comment]
Looks good. docker run for thomx/fr24feed-piaware seems to work fine by pulling it from docker hub now after arm64 was added.

Thanks!

Unable to find image 'thomx/fr24feed-piaware:latest' locally
latest: Pulling from thomx/fr24feed-piaware

**Comment by Thom-x** [comment]
Nice 👍

**Pull request merged**